### PR TITLE
march93

### PR DESCRIPTION
**Pull request by march93**
- https://github.com/clubgamma/RiddlesWorld/issues/23
- Added flutter staggered animation library
- Animated category list using above library
- Added slight delay so animation looks smoother

If this animation looks ok, I can look into adding more for other pages! 

- [x] I have read the Code Of Conduct.
- [x] I have followed all the steps of submission properly.

### CODE REVIEW

**Comment by harshptl14** [comment]
Hey @march93, the animation looks good✨🙌
Should we make a new branch for animation and then merge it with master later?
and yeah, you can start working on another screen!

**Comment by march93** [comment]
@harshptl14 I can keep all the animations in this branch! I'll start updating it with animations for other screens and then we can review it when it's done 😬

**Comment by march93** [comment]
@harshptl14 the animations are all done for every screen, let me know what you think!

**Comment by harshptl14** [comment]
@march93, animations are looking awesome🙌🚀
Can you add animation for [result page also.](https://github.com/clubgamma/RiddlesWorld/blob/master/lib/universal/result.dart)  There is only one result page for all riddles so you need to edit this screen only!
And pull all the changes from the master before giving the PR✨

**Comment by march93** [comment]
@harshptl14 done!

File: lib/categories/Math/MathRiddlelLists.dart [hunk 1]
```diff
@@ -23,135 +19,149 @@ class MathRiddles extends StatefulWidget {
...
-                  name: 'Card Skills',
-                  description:
-                      'A math riddle based on the card deck by @mayankparida',
-                  number: '11',
-                  onPressed: () {
-                    Navigator.of(context).push(MaterialPageRoute(
-                        builder: (ctx) => Math11HomeScreen()));
-                  }),
-
-              ActionCard(
```
Comment (harshptl14) [inline lib/categories/Math/MathRiddlelLists.dart:133]: 12 and 13 number Riddles will be removed if I merge this PR, So can you **Pull these files** and then send the PR again?😶

**Comment by harshptl14** [comment]
Sorry @march93 for the inconvenience 😶👀😓

File: lib/categories/Math/MathRiddlelLists.dart (hunk 1 above)
Comment (march93) [reply lib/categories/Math/MathRiddlelLists.dart:133]: Added those back in, thanks for catching those!

**Review by harshptl14 — APPROVED** [review]
Thanks,@march93, for opening the pull request! 👏

I have reviewed your pull request thoroughly, and it's perfect!

Star ⭐ this project would be great! You can also [tweet](https://twitter.com/share?text=Contributed+to+awesome+projects+at+@club_gamma&url=https://clubgamma.github.io/hacktoberfest) about your contributions🤟
Join the [discord channel of club gamma🙌](https://discord.gg/kjnp6wU)

**Pull request merged**